test_sorting_insertionsort.py

```diff
@@ -0,0 +1,17 @@
+from sorting import insertion_sort
+from random import shuffle
+import pytest
+
+
+def test_randoms():
+    x = range(100)
+    randoms = x[:]
+    shuffle(randoms)
+    insertion_sort(randoms)
+    y = randoms
+    assert x == y
```

Comment:
The reassining of variables back and forth, and making copy of the list is not really needed. You could cut this down to 4 lines like this 
``` def test_random():
    x = range(100)
    shuffle(x)
    insertion_sort(x)
    assert range(100) == x
```